CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
     * Improve validation of `UserCreate.slurm_accounts` (\#1162).
     * Add `timestamp_created` to `WorkflowRead`, `WorkflowDump`, `DatasetRead` and `DatasetDump` (\#1152).
     * Make all dumps in `ApplyWorkflowRead` non optional (\#1175).
+    * Ensure that timestamps in `Read` schemas are timezone-aware, regardless of `DB_ENGINE` (\#1186).
+    * Add and standardise datetime query parameters for admin api, so that they only accept timezone aware datetime (\#1186).
```

Comment:
Also something like: "Add timestamp query-parameters to all `/admin` endpoints. (\#1186)."